### DISCUSSION TITLE
BirdWeather PUC

### DISCUSSION THREAD

**Discussion by dbrianhendrix** [mcguirepr89/BirdNET-Pi#1022] [General]
Just thought I'd share a new product that's out which I was able to use and I got on a pre-sale. It is a VERY neat little device that not only works in a stationary location but also portable. I'll be using it on hikes. I've included the link below:

https://www.kickstarter.com/projects/scribelabs/birdweather-puc/

I've used it in several locations on my property and I'm loving it's ease of use and especially portability!

**Comment by cornerkilling** [comment]
Very excited to try it out. How do you mount it on a wall? I saw a few photos of it mounted on a post but not sure how. Also curious how much you can modify the device. If you needed to self host in the future, could you have it upload recordings locally?

**Reply by waynecarnold** [reply]
It records locally to a microSD card but the actual identification of the birds happens on the server. So you have access to the recordings (and I mean all recording whether they are birds or not) and you would have to analyze them through your own means.  Having said that they are really fun to use. I left mine in the forest for a day and it was fun to see what it heard.

**Comment by blogdemoi** [comment]
Can you easily listen to the recordings in the same way as Birdnet-Pi with a neat webpage?
